recipes/tesseract/all/conanfile.py

```diff
@@ -59,13 +59,13 @@ def layout(self):
         cmake_layout(self, src_folder="src")
 
     def requirements(self):
-        self.requires("leptonica/1.82.0")
+        self.requires("leptonica/1.83.0", transitive_headers=True)
```

Comment:
It's sometimes hard to know the reasoning behind adding this trait, so for posterity: This is needed because of https://github.com/tesseract-ocr/tesseract/blob/9422915eb777f1df063838f9a2b4f72ce3aa6e46/src/tesseract.cpp#L34 (among possibly others)

Reply:
A .cpp file can't be responsible of public visibility of an external header. Instead, you have to look at includes inside public headers of tesseract.

Reply:
leptonica is not a public dependency, why is there this `transitive_headers=True`?

Reply:
It's my fault. 
leptonica transitive_headers is required in test_package.
It is not necessary.
I removed it.